app/assets/javascripts/discourse/app/components/user-menu/profile-tab-content.hbs

```diff
@@ -77,14 +77,7 @@
       <span class="item-label">
         {{#if this.isInDoNotDisturb}}
           <span>{{i18n "do_not_disturb.label"}}</span>
-          <span
-            title={{this.doNotDisturbDateTitle}}
-            data-time={{this.doNotDisturbDateTime}}
-            data-format="tiny"
-            class="relative-date"
-          >
-            {{this.doNotDisturbDateContent}}
-          </span>
```

Comment:
Can we remove the `doNotDisturbDateTitle` and `doNotDisturbDateContent` getters in profile-tab-content.js?

Reply:
We need to remove them for sure. I removed `doNotDisturbDateTitle`, but didn't notice `doNotDisturbDateContent`, I've removed it too now.